pkg/ovsdb/service_test.go

```diff
@@ -5,23 +5,25 @@ import (
 	"fmt"
 	"testing"
 
-	"github.com/ibm/ovsdb-etcd/pkg/common"
-
 	"github.com/stretchr/testify/assert"
+
+	"github.com/ibm/ovsdb-etcd/pkg/common"
 )
 
-const baseTransact string = "../../tests/data/transact/"
+const BaseData string = "../../tests/data/"
```

Comment:
this is only going to be used internally ... anyone using `BaseData` from a different location will fail

Reply:
ok